src/index.js

```diff
@@ -38,13 +35,7 @@ async function fetchPlatforms () {
         throw new Error(AUTH_FAILED_ERROR);
 
     const host = process.env.SAUCE_API_HOST || 'us-west-1.saucelabs.com';
-    const response = await request(`https://api.${host}/rest/v1.1/info/platforms/all`,
-        {
-            'auth': {
-                'user': process.env['SAUCE_USERNAME'],
-                'pass': process.env['SAUCE_ACCESS_KEY']
-            }
-        });
+    const response = await getRequest(`https://api.${host}/rest/v1.1/info/platforms/all`, process.env['SAUCE_USERNAME'], process.env['SAUCE_ACCESS_KEY']);
```

Comment:
I suggest sending request params as Object. Fewer args - more straightforward working with them and easier researching the function.